src/openmrs-esm-error-handling.test.js

```diff
@@ -0,0 +1,27 @@
+jest.mock("./openmrs-esm-error-handling");
+const { reportError } = jest.requireActual("./openmrs-esm-error-handling");
+
+jest.useFakeTimers();
+describe("error handler", () => {
+  it("transfrom the input in valid error object if it is not already an error obejct", () => {
+    expect(() => {
+      reportError("error");
+      jest.runAllTimers();
+    }).toThrow("error");
+
+    expect(() => {
+      reportError({ error: "error" });
+      jest.runAllTimers();
+    }).toThrow('Object thrown as error: {"error":"error"}');
+
+    expect(() => {
+      reportError(null);
+      jest.runAllTimers();
+    }).toThrow("Object thrown as error: null");
```

Comment:
This test is incorrect. It should verify that the thrown error is `'null' was thrown as an error`. Please implement the fix I suggested in https://github.com/openmrs/openmrs-esm-error-handling/pull/7/files#r334546381 and then update this test to be correct.